contributing/C_API.md

```diff
@@ -66,17 +74,17 @@ we will `reinterpret_cast` to/from.
 
 ### Alternatives
 
-Alternative schemes exist for adding type safety to opaque pointer 
```

Comment:
[fix] Sorry, nit pick, but can we split the whitespace changes out into their own commit at the top of the PR?

Reply:
Whoops, missed one. The find-replace also removed trailing whitespace - I reverted most changes, but obviously missed some.